SDL2pp/Font.cc

```diff
@@ -149,7 +153,7 @@ Rect Font::GetGlyphRect(Uint16 ch) const {
 	int minx, maxx, miny, maxy;
 	if (TTF_GlyphMetrics(font_, ch, &minx, &maxx, &miny, &maxy, nullptr) != 0)
 		throw Exception("TTF_GlyphMetrics");
-	return Rect(minx, miny, maxx - minx + 1, maxy - miny + 1);
+	return Rect(minx, miny, maxx - minx, maxy - miny);
```

Comment:
You sure about this? SDL_Rect logic is a bit non-straightforward:
```
 0  1  2  3  4  5
[ ][ ][ ][ ][ ][ ]
[ ][X][X][X][X][ ]
[ ][X][o][o][X][ ]
[ ][X][X][X][X][ ]
[ ][ ][ ][ ][ ][ ]
```
here ```minx = 1```, ```maxx = 4```, and ```width = 4 = maxx - minx + 1```